nav-hinting.md

```diff
@@ -131,6 +131,7 @@ The NH Engine has a configuration file defined below. This defines parameters fo
   "engineId": "hinting",
   "engineFactory": "com.actionml.templates.nh.NavHintingEngine",
   "algorithm":{
+    "journeyTTL": 90, // number of days, 90 is default
```

Comment:
This is to set the TTL on the journey's collection, not used yet but the setting is available in the config params